packages/forms-web-app/src/pages/projects/section-51/index/_components/results.njk

```diff
@@ -29,4 +29,4 @@
 			</li>
 		{% endfor %}
 	</ul>
-{% endmacro %}
+{% endmacro %}
```

Comment:
Could save an extra file commit by undo the formatting

Reply:
How would I do this? Everytime I save it adds this formatting which should be coming from eslint.